velox/functions/sparksql/Size.cpp

```diff
@@ -30,9 +30,13 @@ struct Size {
   template <typename TInput>
   FOLLY_ALWAYS_INLINE void initialize(
       const std::vector<TypePtr>& /*inputTypes*/,
-      const core::QueryConfig& config,
-      const TInput* input /*input*/) {
-    legacySizeOfNull_ = config.sparkLegacySizeOfNull();
+      const core::QueryConfig& /*config*/,
+      const TInput* /*input*/,
+      const bool* legacySizeOfNull) {
+    if (legacySizeOfNull == nullptr) {
+      VELOX_USER_FAIL("Constant legacySizeOfNull is expected.");
+    }
+    legacySizeOfNull_ = *legacySizeOfNull;
```

Comment:
> Spark size function's legacySizeOfNull is specified either by other functions
(e.g., array_size function) or by configuration.

Are we removing the specification from query config, and shall we keep it according to above description?

Reply:
Yes, we need remove this config. Just updated description to clarify.